.ui/library/flAlert.cpx

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
we switched to native alert iirc. Could you remove this?

Reply:
done.